tools/kill_gmt.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/env bash
+#!/bin/bash
```

Comment:
Why this change?

Using /usr/env is tremendously helpful if your bash is not installed in /bin and / or you have an overlayed bash (as I have on macOS which comes through brew that places it in /opt/homebrew/...)

Did you run into errors with this in this particulate PR?

Reply:
Yes, I run into an error while implementing this PR (or more specifically looking into the issue of the time travel mode 3, see https://github.com/green-coding-solutions/green-metrics-tool/issues/1292).
Error:

```sh
$ ./tools/kill_gmt.sh 
bash: ./tools/kill_gmt.sh: cannot execute: required file not found
```

`kill_gmt.sh` is the only shell script in the repo that was using the shebang `#!/usr/env bash`. All the others are using `#!/bin/bash`. Besides fixing the error I also wanted to have more consistency.

Reply:
I reverted the shebang change in this PR and added a new one: https://github.com/green-coding-solutions/green-metrics-tool/pull/1299